yt/documentation/en/toc.yaml

```diff
@@ -279,7 +279,7 @@ items:
                    href: user-guide/data-processing/spyt/API/spyt-java.md
                  - name: SPYT in Python
                    href: user-guide/data-processing/spyt/API/spyt-python.md
-                 - name: SPYT in Jyputer
+                 - name: SPYT in Jyputyr
```

Comment:
It is spelled as "Jupyter" (https://en.wikipedia.org/wiki/Project_Jupyter) :)

Reply:
@gritukan that's why I changed Jyputer to Jyputyr